core-bundle/src/Security/Voter/MemberGroupVoter.php

```diff
@@ -35,22 +45,22 @@ protected function voteOnAttribute(string $attribute, mixed $subject, TokenInter
         $subject = array_filter($subject, static fn ($val) => (string) (int) $val === (string) $val);
 
         if (!$subject) {
-            return false;
+            return self::ACCESS_DENIED;
         }
 
         $user = $token->getUser();
 
         if (!$user instanceof FrontendUser) {
-            return \in_array(-1, array_map('intval', $subject), true);
+            return \in_array(-1, array_map('intval', $subject), true) ? self::ACCESS_GRANTED : self::ACCESS_DENIED;
         }
 
         $groups = StringUtil::deserialize($user->groups, true);
 
         // No groups assigned
         if (empty($groups)) {
-            return false;
+            return self::ACCESS_DENIED;
```

Comment:
Shouldn‘t this be `self::ACCESS_ABSTAIN` as all our voters either deny or abstain? Also applies to lines 54 and 64.